lib/order.rb

```diff
@@ -0,0 +1,78 @@
+require 'csv'
+require 'customer'
+require 'pry'
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+  
+  STATUS = [:pending, :paid, :processing, :shipped, :complete]
```

Comment:
Style: collections should have plural names.

```suggestion
  STATUSES = [:pending, :paid, :processing, :shipped, :complete]
```